src/web/EmployerFavouritesWeb/Commands/SaveApprenticeshipFavouriteCommand.cs

```diff
@@ -0,0 +1,11 @@
+using MediatR;
+
+namespace DfE.EmployerFavourites.Web.Commands
+{
+    public class SaveApprenticeshipFavouriteCommand : IRequest
+    {
+        public string UserId { get; set; }
+        public string ApprenticeshipId { get; set; }
+        public int? Ukprn { get; set; }
```

Comment:
Would it make more sense for this this to be a list of ukprn's instead of singular? otherwise when we have multiple training providers we will need to call this multiple times? unless you see this command only being used for the current implementation without a basket

Reply:
Was only coded to support the current implementation of a single one of each.